tests/ballerina-spec-conformance-tests/S05_values_types_variables/tests/implicit_initial_value.bal

```diff
@@ -118,3 +118,15 @@ function testImplicitInitialValueOfJSONType() {
     jsonArray[1] = 3;
     test:assertEquals(jsonArray[0], (), msg = "expected implicit initial value of json type to be ()");
 }
+
+public type QuxObject object {
+    public string fooFieldOne;
+
+    public function __init(string fooFieldOne) {
```

Comment:
Shall we refactor this object so that it has an IIV and test it with a new function `testImplicitInitialValueOfObjectType()`?

Reply:
I don't think we can do that at the moment, we need to wait for @pubudu91 s PR to be merged in order to get a IIV of an object. Lets add a deviation test, WDYT?

Reply:
Ah yes we'll add a deviation test in that case (y)